.github/workflows/Perf_Env_Build_Test_CI.yml

```diff
@@ -288,7 +288,7 @@ jobs:
           ELASTICSEARCH_USER: ${{ secrets.PERF_ELASTICSEARCH_USER }}
           ELASTICSEARCH_PASSWORD: ${{ secrets.PERF_ELASTICSEARCH_PASSWORD }}
         run: |
-          python "$GITHUB_WORKSPACE"/benchmark_runner/grafana/update_versions_main_libsonnet.py
+          PYTHONPATH=. python "$GITHUB_WORKSPACE"/benchmark_runner/grafana/update_versions_main_libsonnet.py
```

Comment:
PYTHONPATH=. refers to the current directory.  If you're cd'ed into /tmp, say, everything will be referenced relative to /tmp.

A better way to write it would be

```
cd "$GITHUB_WORKSPACE" && PYTHONPATH=. benchmark_runner/grafana/update_versions_main_libsonnet.py
```

Reply:
Agreed, fixed.